pymobiledevice3/services/installation_proxy.py

```diff
@@ -124,20 +165,35 @@ def install_from_local(self, package_path: Path, cmd: str = 'Install', options:
             else:
                 self.upload_ipcc_as_folder(package_path, afc)
 
-        self.service.send_plist({'Command': cmd,
-                                 'ClientOptions': options,
-                                 'PackagePath': TEMP_REMOTE_IPCC_FOLDER if ipcc_mode
-                                 else TEMP_REMOTE_IPA_FILE})
+        self.send_package(cmd, options, handler, ipcc_mode, *args)
+
+    def send_package(self, cmd: str, options: Optional[dict], handler: Callable, ipcc_mode: bool = False, *args):
+        self.service.send_plist({
+            'Command': cmd,
+            'ClientOptions': options,
+            'PackagePath': (
+                TEMP_REMOTE_IPCC_FOLDER if ipcc_mode
+                else TEMP_REMOTE_IPA_FILE
+            )
+        })
 
         self._watch_completion(handler, ipcc_mode, args)
 
-    def upload_ipcc_as_folder(self, file: Path, afc_client: AfcService) -> None:
+    def upload_ipcc_as_folder(self, file: Union[Path, bytes], afc_client: AfcService) -> None:
```

Comment:
restore this function behavior to the old one. the bytes objects can already be convered to file in `install_from_bytes()`

Reply:
i don't want an api function acting differently depending in its supplied arguments. if it performs another logic it should be in its own api. the expected `file` argument can remain `Path`